test/jdk/ProblemList-fips.txt

```diff
@@ -17,3 +17,1207 @@
 # You should have received a copy of the GNU General Public License version
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
 # ===========================================================================
+
+#
+# Exclude tests list from sanity.openjdk
+#
+# SunJCE and SunRsaSign related
+
+com/sun/crypto/provider/Cipher/AEAD/Encrypt.java https://github.com/ibmruntimes/openj9-openjdk-jdk17/issues/116 linux-all
```

Comment:
Just want to double-check, `linux-all` means to exclude the subtest on all linux based platforms (not just x/p/zLinux). Is it intended?

Reply:
This is for NSS FIPS 140-2, so only the Redhat Linux platforms need to use this exclude list. Right now, the Redhat linux NSS FIPS supports the x/p/z linux. So, if we don't use the `linux-all` flag, do we have any other specific flag only for x/p/z linux?

Reply:
> if we don't use the linux-all flag, do we have any other specific flag only for x/p/z linux?

yes, you can use `linux-x64,linux-ppc64le,linux-s390x`

> Or use the linux-all flag is good enough?

I just want to point out, `linux-all` applies to all Linux based platforms, not just x/p/zLinux. For example, if linux-aarch64 platform is enabled in the future, it will be excluded. If this behavior is intended, then we should use `linux-all`.

Reply:
Updated to use "linux-x64,linux-ppc64le,linux-s390x" for these 3 NSS FIPS supported platforms. Please help to review.